### ISSUE TITLE
Enable login for personal microsoft accounts

### ISSUE THREAD

**Issue by andifandrich** [Azure-Samples/active-directory-dotnet-native-aspnetcore-v2#66]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [X] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> Tried the "2 Web API now calls Microsoft Graph" with personal MS accounts, but unfortunately this produces an error in line 408 of class TokenAcquisition.
AADSTS70000: The request was denied because one or more scopes requested are unauthorized or expired. The user must first sign in and grant the client application access to the requested scope.
Of course my app registration allows those to sign in and of course all necessary scopes are granted.

### Any log messages given by the failure
>

### Expected/desired behavior
No error

### OS and Version?
Deployed to Azure app service

### Versions
>

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by jmprieur** [comment]
@andifandrich 

Did you try: https://github.com/Azure-Samples/active-directory-dotnet-native-aspnetcore-v2/tree/master/3.-Web-api-call-Microsoft-graph-for-personal-accounts ?
it works with MSA (but this might not be what you need)

There is currently a limitation that "Known client applications" don't work yet with personal MS Accounts .It's under development (I hope to have it before the end of the month).

**Comment by andifandrich** [comment]
Thanks for your quick response and yes I also tried the mentioned sample. In fact my scenario is a Web API used by a JavaScript frontend and also by a WPF application, so both samples fit.
I know about the limitations with personal MS accounts (at least I thought I did). But I also thought it'd be possible to get the most simple thing e.g. login with a personal account. Everything works perfectly well with O365 accounts.
As you're talking about an update hopefully due the end of the month, I think I'll just wait and try again when it is available.

Thanks for your effort (and of course for all the samples). I rellay appreciate it.
Andi Fandrich
